Twispay WooCommerce Plugin/twispay/includes/twispay-processor.php

```diff
@@ -94,6 +94,8 @@
                     /* , 'tags' => [] */
                     ];
 
+        error_log("customer=" . print_r($customer), true);

```

Comment:
Please check the line, `print_r` will output the content without the second parameter set to bool true.

Reply:
I removed the line as it was a debug leftover.